spec/factories/profile_fields.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :profile_field do
-    profile_field_group
+    profile_field_group { FactoryBot.create(:profile_field_group) }
```

Comment:
Maybe we can use `association` instead as outlined [in the documentation](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#explicit-definition).

```suggestion
    association: :profile_field_group
```